data/meeting_minutes.yml

```diff
@@ -757,6 +757,30 @@ items:
           year: 2018
 
    steering_committee:
+          -
```

Comment:
This line's spacing is not consistent with the other lines that denote a new meeting minute document (Line 764 and 768 are also examples of this). If you match the number of spaces on those lines on this one, it should pass. The storage format used for this is a bit dependent on spacing.

Reply:
Should be all fixed now.